pkg/controller/directimagestreammigration/migrate.go

```diff
@@ -44,6 +46,8 @@ func (r *ReconcileDirectImageStreamMigration) migrate(imageStreamMigration *miga
 		if errors.IsConflict(errorutil.Unwrap(err)) {
```

Comment:
Same question, should we create an issue for this?

Reply:
Would like to understand what other kind of errors we would want to handle here, see https://github.com/konveyor/mig-controller/pull/992#discussion_r595439193